scripts/prepare-release.sh

```diff
@@ -30,14 +30,14 @@ create_new_csv() {
 
   if [[ -z "$PREVIOUS_VERSION" ]]
     then
-      operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name "$OLM_TYPE" --csv-channel=rhmi --update-crds
+      operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name "$OLM_TYPE" --csv-channel=rhmi --update-crds --make-manifests=false
```

Comment:
have we updated the version of operator-sdk in the pipelines now? Last time we tried to add this flag the operator-sdk version in the pipelines was too old to understand this parameter.

Reply:
@laurafitzgerald

Reply:
Ah, @CathalOConnorRH confirmed this is just for the sake of verifying the PR locally, and will not be merged.